src/stryker-net/reporters.pug

```diff
@@ -0,0 +1,73 @@
+extends ../layout/section-layout.pug
+
+block content
+    :markdown-it
```

Comment:
Similar as above. There's already some configuration documentation in the Stryker-CLI readme. I would suggest keeping it in one place (whether that's the website or in the project)